docs/api/networksocket/SocketStats.md

```diff
@@ -0,0 +1,21 @@
+## SocketStats
+
+Use the SocketStats class to read the network socket statistics. SocketStats Class object is created by "InternetSocket" class when statistics are enabled by `nsapi.socket-stats-enable` config option.
+
+The `SocketStats` class provides the static function `mbed_stats_socket_get_each` to get the network socket information for all sockets both currently in use and closed. Using this function you can know how many sockets are open, destination IP addresses, the amount of data sent and received in bytes, the type of socket and the timestamp of recent "socket state" changes recorded in `last_change_tick`.
+
+Socket Statistics are stored in intenal data buffer and maintained for closed sockets as well. In case internal data buffer is full, socket closed with oldest timestamp will be over-written. Maximum amount of sockets statistics stored can be configured by `nsapi.socket-stats-max-count` config option.
+
+### SocketStats class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/)
```

Comment:
@AnotherButler - Network stats PR https://github.com/ARMmbed/mbed-os/pull/8592 is merged, and doxygen for new Class will be generated from features/netsocket/SocketStats.h. Please help me with the link here.

Reply:
http://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/class_socket_stats.html